job-dsls/jobs/dailyBuild_pipeline.groovy

```diff
@@ -345,15 +345,22 @@ matrixJob("${folderPath}/daily-build-${baseBranch}-jbpmTestContainerMatrix") {
 // +++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
 //  run additional test: kieWbTestsMatrix
 def kieWbTest='''#!/bin/bash -e
-echo "KIE version $kieVersion"
-# wget the tar.gz sources
+echo "KIE version $kieVersion - kie-wb-distributions"
 wget -q http://\${LOCAL_NEXUS_IP}:8081/nexus/content/repositories/kieAllBuild-$baseBranch/org/kie/kie-wb-distributions/$kieVersion/kie-wb-distributions-$kieVersion-project-sources.tar.gz -O sources.tar.gz
 tar xzf sources.tar.gz
+rm sources.tar.gz
 mv kie-wb-distributions-$kieVersion/* .
-rmdir kie-wb-distributions-$kieVersion'''
+rmdir kie-wb-distributions-$kieVersion
+
+echo "KIE version $kieVersion - kie-wb-common"
+wget -q http://\\${LOCAL_NEXUS_IP}:8081/nexus/content/repositories/kieAllBuild-$baseBranch/org/kie/kie-wb-common/$kieVersion/kie-wb-common-$kieVersion-project-sources.tar.gz -O sources.tar.gz
+tar xzf sources.tar.gz
+rm sources.tar.gz
+mv kie-wb-common-$kieVersion/* .
+rmdir kie-wb-common-$kieVersion'''
 
 matrixJob("${folderPath}/daily-build-${baseBranch}-kieWbTestsMatrix") {
-    description("This job: <br> - Runs the KIE Server integration tests on mutiple supported containers and JDKs <br> IMPORTANT: Created automatically by Jenkins job DSL plugin. Do not edit manually! The changes will get lost next time the job is generated. ")
+    description("This job: <br> - Runs the KIE wb integration tests on mutiple supported containers and JDKs <br> IMPORTANT: Created automatically by Jenkins job DSL plugin. Do not edit manually! The changes will get lost next time the job is generated. ")
```

Comment:
just an optional thing, do we use uppercase shortcut here? s/wb/WB/

Reply:
Applied